RecoPPS/Local/plugins/CTPPSPixelRecHitProducer.cc

```diff
@@ -20,22 +21,27 @@ void CTPPSPixelRecHitProducer::produce(edm::Event &iEvent, const edm::EventSetup
   edm::Handle<edm::DetSetVector<CTPPSPixelCluster> > rpCl;
   iEvent.getByToken(tokenCTPPSPixelCluster_, rpCl);
 
+  edm::ESHandle<PPSPixelTopology> thePixelTopology = iSetup.getHandle(pixelTopologyToken_);
+
   edm::DetSetVector<CTPPSPixelRecHit> output;
 
   // run reconstruction
   if (!rpCl->empty())
-    run(*rpCl, output);
+    run(*rpCl, output, *thePixelTopology);
 
   iEvent.put(std::make_unique<edm::DetSetVector<CTPPSPixelRecHit> >(output));
+
+  //  std::cout << (*thePixelTopology) << std::endl;
```

Comment:
this might be removed

Reply:
Sure. Removed. Waiting for more comments before pushing the code.